charts/astronomer/values.yaml

```
@@ -328,7 +330,9 @@ houston:
     #   - 2.0.0-alpine3.10-onbuild
     #   channel: stable
 
+  runtimeReleasesConfigMapName: ~
   RuntimeReleasesConfig:
```

This should be lowercase R
```suggestion
  runtimeReleasesConfig:
```